src/main/java/net/shipilev/ForkJoin.java

```diff
@@ -16,9 +16,9 @@
 import java.util.concurrent.TimeUnit;
 
 @State(Scope.Benchmark)
-@Warmup(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
-@Fork(5)
+@Warmup(iterations = 3, time = 1, timeUnit = TimeUnit.MILLISECONDS)
```

Comment:
To set constant iteration numbers (3 for warming and 5 for result stat). 
If time is not set then JMH iterate several times until exceeds 1 second.